docs/learn/learn-runtime-upgrades.md

```diff
@@ -56,56 +45,55 @@ Although upgrading your nodes is generally not necessary to follow an upgrade, w
 following the Polkadot releases and upgrading promptly, especially for high priority or critical
 releases.
 
+## Runtime Upgrades for Various Users
+
 ### For Infrastructure Providers
 
-Infrastructure services including but not limited to the following:
+Infrastructure services include but are not limited to the following:
 
 - [Validators](../maintain/maintain-guides-how-to-upgrade.md)
 - API services
 - Node-as-a-Service (NaaS)
-- General infrastructure management (e.g. block explorers, exchanges)
+- General infrastructure management (e.g. block explorers, custodians)
+- [Wallets](../build/build-wallets.md)
 
 For validators, keeping in sync with the network is key. At times, upgrades will require validators
-to upgrade within a specific time frame to ensure continued sync with their node. It is essential to
-check the release notes, starting with the upgrade priority and acting accordingly.
+to upgrade their clients within a specific time frame, for example if a release includes breaking
+changes to networking. It is essential to check the release notes, starting with the upgrade
+priority and acting accordingly.
 
-For general infrastructure providers, aside from following the Polkadot releases and upgrading in a
-timely manner, somethings to keep an eye out on is updating your parsing logic and acknowledging
-changes to the available RPC clients, such as the
+General infrastructure providers, aside from following the Polkadot releases and upgrading in a
+timely manner, should monitor changes to runtime events and auxilliary tooling, such as the
```

Comment:
"upgrading in a timely manner, should monitor" -> "upgrading in a timely manner should monitor"